apptail/config.go

```diff
@@ -12,10 +12,11 @@ type leakyBucketConfig struct {
 }
 
 type Config struct {
-	MaxRecordSize     int    `json:"max_record_size"`
-	MaxLinesPerSecond int64  `json:"max_lines_per_second"`
-	MaxLinesBurst     uint16 `json:"max_lines_burst"`
-	FileSizeLimit     int64  `json:"read_limit"`
+	MaxRecordSize      int    `json:"max_record_size"`
+	MaxLinesPerSecond  int64  `json:"max_lines_per_second"`
+	MaxLinesBurst      uint16 `json:"max_lines_burst"`
+	FileSizeLimit      int64  `json:"read_limit"`
+	RetentionPerSecond int64  `json:"retention_per_second"`
```

Comment:
Yeah, that name is a bit confusing.  Something along the lines of `PersistPositionIntervalSeconds` perhaps?  "Retention" is pretty vague, and it's an interval, not a frequency ("per second").